src/content/docs/integrations/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx

```diff
@@ -32,674 +34,258 @@ To install the Kubernetes integration using Helm, you will need your New Relic a
 1. Find and copy your [New Relic license key](/docs/accounts/install-new-relic/account-setup/license-key).
 2. Find the name of your cluster with this command:
 
-   ```
-   kubectl config current-context
-   ```
-3. Make sure that `kube-state-metrics` is installed on your machine:
-
-   ```
-   kubectl get deployment --all-namespaces | grep kube-state-metrics
-   ```
-
-   If it's not installed, follow the [instructions in the kube-state-metrics GitHub repo](https://github.com/kubernetes/kube-state-metrics#kubernetes-deployment) to install it.
+```
+kubectl config current-context
+```
 
 ## Install Kubernetes integration with Helm [#install-k8-helm]
 
 Follow the instructions for your version of Helm:
 
 <CollapserGroup>
   <Collapser
-    className="freq-link"
-    id="install-helm3"
-    title="Helm 3"
+      className="freq-link"
+      id="install-helm3"
+      title="Helm 3 using nri-bundle (recommended)"
   >
-    To install using Helm 3:
+
+    New Relic has several charts for the different components which offer different features for the platform:
+
+    - [`newrelic-infrastructure`](https://github.com/newrelic/helm-charts/tree/master/charts/newrelic-infrastructure):
+    Contains the main Kubernetes integration and the infrastructure agent. This is the bare minimum needed to get data from
```

Comment:
This is not 100% correct, kube-events, pomi do not need the infrastructure to be installed. it is the main component, but still it is not "required" to have the different components working